### PR TITLE
Upgrade to ProximalAlgorithms v0.5

### PR DESCRIPTION
**Pull request by dhanak**
* drop support for ForwardBackward optimizer;
* add support for PANOCplus;
* make PANOCplus the default optimizer;
* stabilize unstable unit test.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -13,10 +13,10 @@ RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 
 [compat]
 AbstractOperators = "0.1 - 0.2"
```
Comment (lostella) [inline Project.toml:15]:
```suggestion
AbstractOperators = "0.3"
```

**Review by lostella — CHANGES_REQUESTED** [review]
Thanks @dhanak! I guess we can bump the dependency on AbstractOperators too.

I’m not entirely sure why we need to drop support for ForwardBackward?

File: Project.toml (hunk 1 above)
Comment (dhanak) [reply Project.toml:15]: Sure. In fact, I requested a version bump on AbstractOperators, I'm glad it went through so quickly.

**Comment by dhanak** [comment]
> I’m not entirely sure why we need to drop support for ForwardBackward?

ProximalAlgorithms dropped support for an external `A` matrix, `ForwardBackward` is now just `f(x) + g(x)`, so the transformations in StructuredOptimzation don't work with FB anymore. There are the `DouglasRatchford` and the `DRLS` solvers, which could be added to the suite, though.

**Comment by dhanak** [comment]
> I’m not entirely sure why we need to drop support for ForwardBackward?

ProximalAlgorithms dropped support for an external `A` matrix, `ForwardBackward` is now just `f(x) + g(x)`, so the transformations in StructuredOptimzation don't work with FB anymore. There are the `DouglasRatchford` and the `DRLS` solvers, which could be added to the suite, though.

**Comment by dhanak** [comment]
Please note that I won't be available in the next 10 days or so.

**Review by lostella — APPROVED** [review]
Thanks @dhanak!

**Pull request merged**